schema/core/reusable/param_int.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
maybe let's have a folder `parameters/` and nest integer, number, and string inside it?

Reply:
agreed. They are thematically and structurally very similar.